ios/Mattermost.xcodeproj/project.pbxproj

```diff
@@ -1146,6 +1203,7 @@
 			isa = XCBuildConfiguration;
 			buildSettings = {
 				BUNDLE_LOADER = "$(TEST_HOST)";
+				DEVELOPMENT_TEAM = "";
```

Comment:
Sure didn't, was doing some device testing and these changes snuck their way in. I'll change them back.